runtime.md

```diff
@@ -54,17 +56,28 @@ The lifecycle describes the timeline of events that happen from when a container
 3. Once the container is created additional actions MAY be performed based on the features the runtime chooses to support.
    However, some actions might only be available based on the current state of the container (e.g. only available while it is started).
 4. Runtime's `start` command is invoked with the unique identifier of the container.
-   The runtime MUST run the user-specified code, as specified by [`process`](config.md#process-configuration).
-5. The container's process is stopped.
+5. The [prestart hooks](config.md#prestart) MUST be invoked by the runtime.
```

Comment:
On IRC, @mrunal [suggested switching steps 4 and 5][1], so the `pre-start` hooks fire at the end of `create` instead of the beginning of `start`.  I think that would be surprising considering the naming (621bdb4 talks about a possible `postcreate` anchor in that position).  But we already have the confusing `poststop` running after `delete` (621bdb4 talks about this too).

Personally, I'd rather drop hooks (#483).  But if we keep them for backwards compat, I can see that updating consumers to go from runtime-executed hooks to orchestrator executed hooks would be a largish change.  However, renaming config fields (`prestart` → `postcreate`) seems like a trivial change, and I'd much rather force pre-1.0 users to pay that cost now (they understood pre-1.0 was unstable) than have to explain to all new users why our hook names make no sense ;).

[1]: http://ircbot.wl.linuxfoundation.org/eavesdrop/%23opencontainers/%23opencontainers.2016-08-17.log.html#t2016-08-17T22:52:49